tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\LanguageConstruct;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class FunctionToConstantFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     * @param null|array  $config
+     *
+     * @dataProvider provideTestCases
+     */
+    public function testFix($expected, $input = null, array $config = null)
+    {
+        if ($config) {
+            $this->fixer->configure($config);
+        }
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideTestCases()
+    {
+        return array(
+            'Minimal case, alternative casing, alternative statement end.' => array(
+                '<?php echo PHP_VERSION?>',
+                '<?php echo PHPversion()?>',
+            ),
+            'With embedded comment.' => array(
+                '<?php echo PHP_VERSION/**/?>',
+                '<?php echo phpversion(/**/)?>',
+            ),
+            'With white space.' => array(
+                '<?php echo PHP_VERSION      ;',
+                '<?php echo phpversion  (  )  ;',
+            ),
+            'With multi line whitespace.' => array(
+                '<?php echo
+                PHP_VERSION
+                '.'
+                '.'
+                ;',
+                '<?php echo
+                phpversion
+                (
+                )
+                ;',
+            ),
+            'Global namespaced.' => array(
+                '<?php echo \PHP_VERSION;',
+                '<?php echo \phpversion();',
+            ),
+            'Wrong number of arguments.' => array(
+                '<?php phpversion($a);',
+            ),
+            'Wrong namespace.' => array(
+                '<?php A\B\phpversion();',
+            ),
+            'Class creating.' => array(
+                '<?php new phpversion();',
+            ),
+            'Class static method call.' => array(
+                '<?php A::phpversion();',
+            ),
+            'Class method call.' => array(
+                '<?php $a->phpversion();',
+            ),
+            'Overridden function.' => array(
+                '<?php if (!function_exists("phpversion")){function phpversion(){}}?>',
+            ),
+            'phpversion only' => array(
+                '<?php echo PHP_VERSION; echo php_sapi_name(); echo pi();',
+                '<?php echo phpversion(); echo php_sapi_name(); echo pi();',
+                array(
+                    'phpversion' => true,
+                    'php_sapi_name' => false,
```

Comment:
there isn't, there was some confusion about the new config structure, will be solved in another way